### PR TITLE
Step1 introduce yourself flow

### PR DESCRIPTION
**Pull request by alexchoiweb**
Fixes issue #126 

### Used state and conditional rendering to implement user flow and interaction styling
-Used some logic in the conditional rendering in the jsx, let me know if this is not optimal (line 75, 90)
-Moved `Popup` info from `Form` to `Step1` as I needed to manipulate it inside `Step1`
-Added a optional prop "defaultValue" to `Input` to keep already-inputted values when user goes back to edit from `TextPreview`

https://user-images.githubusercontent.com/35116846/109405716-3765c000-7928-11eb-8745-34277fc7554d.mov

### CODE REVIEW

File: react/client/src/components/RadioGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  label,
+  activeRadio,
+  handleChange,
+  inputName,
+}: RadioButtonProps) => {
+  return (
+    <div>
+      <Radio
+        value={label}
+        color="primary"
```
Comment (sarahyaepark) [inline react/client/src/components/RadioGroup.tsx:21]: Not sure if it's still necessary, but the figma has the color of the radio button as the bright teal color

**Review by sarahyaepark — COMMENTED** [review]
The flow logic looks good to me!

File: react/client/src/components/RadioGroup.tsx (hunk 1 above)
Comment (alexchoiweb) [reply react/client/src/components/RadioGroup.tsx]: Nice, Good eye, I was thinking we could finish with the styling in another Issue/PR and have this one just be for logic but I have made that change.

**Review by aahvocado — APPROVED** [review]
Also looks good to me!

**Pull request merged**